content-src/lib/ActionManager.js

```diff
@@ -41,7 +41,7 @@ class ActionManager {
       throw new Error("Looks like your action definition does not return an object.");
     }
     if (!action.type) {
-      throw new Error(`You must define a type for an action.`);
+      throw new Error("You must define a type for an action.");
```

Comment:
Turns out ESLint doesn't like using the backtick strings if you don't have any nested variables. So clever.